src/stateless/slider.vue

```diff
@@ -0,0 +1,597 @@
+<template>
+    <div class="slider-container" :class="Object.assign(sizeClass(), themeClass())">
+
+        <div class="slider-header">
+            {{ label }}
+        </div>
+
+        <div class="slider-unit">
+
+            <div class="slider-input">
+                <input-element
+                    :type="inputType"
+                    :size="size"
+                    :disabled="disabled"
+                    :value="currentValue"
+                    :step="step"
+                    :min-value="0"
+                    :max-value="999"
```

Comment:
Use minValue and maxValue props

Reply:
Weirdly, it should be like this

Reply:
Why exactly 0 and 999? I also see you're checking if min and max props are between 0 and 999 in the `created` hook. Why can't max be greater than 999?